src/commands/IssuesCommand.ts

```diff
@@ -0,0 +1,63 @@
+import { Message, MessageEmbed } from "discord.js";
+import Command from "../abstracts/Command";
+import GitHubService from "../services/GitHubService";
+import { EMBED_COLOURS } from "../config.json";
+import GitHubIssue from "../interfaces/GitHubIssue";
+
+class IssuesCommand extends Command {
+	constructor() {
+		super(
+			"issues",
+			"Shows all the issues of the given repository."
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		const embed = new MessageEmbed();
+
+		const hasNoArgs = !args || typeof args[0] === "undefined";
+		const notCorrectFormat = !args[0]?.includes("/");
+
+		if (hasNoArgs || notCorrectFormat) {
+			embed.setTitle("Error");
+			embed.setDescription("You must provide a username and repo from GitHub.");
+			embed.addField("Correct Usage", "?issues <username>/<repository>");
+			embed.setColor(EMBED_COLOURS.ERROR);
+		} else {
+			const [user, repoName] = args[0].split("/");
+
+			try {
+				const GitHub = GitHubService.getInstance();
+				const resIssues = await GitHub.getIssues(user, repoName);
+				const resRep = await GitHub.getRepository(user, repoName);
+
+				if (resIssues.length > 0) {
+					embed.setTitle(`GitHub Issues: ${user}/${repoName}`);
+					embed.setDescription(`${resRep.description}\n\n[View Issues on GitHub](${resRep.url}/issues) - [Create one](${resRep.url}/issues/new)`);
+
+					resIssues.forEach((issue: GitHubIssue) => {
+						const diff = Math.abs(Date.now() - issue.created_at.getTime());
+						const days = Math.ceil(diff / (1000 * 3600 * 24));
```

Comment:
This could be broken into a helper `getDaysBetweenDates(date1: Date, date2: Date): number`. Could be useful in the future.